ETSGlobalLogBundle.php

```diff
@@ -5,15 +5,13 @@
 namespace ETSGlobal\LogBundle;
 
 use ETSGlobal\LogBundle\DependencyInjection\CompilerPass\HttpClientPass;
-use ETSGlobal\LogBundle\DependencyInjection\CompilerPass\LoggerAwarePass;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 class ETSGlobalLogBundle extends Bundle
 {
     public function build(ContainerBuilder $container): void
     {
-        $container->addCompilerPass(new LoggerAwarePass());
```

Comment:
:+1: not needed since Symfony 4.2 https://github.com/symfony/symfony/commit/afda3c8844a78b47f4229793cc351f637d3dfae1